core/vm/contracts.go

```diff
@@ -190,7 +190,6 @@ var PrecompiledCeloContractsCel2 = map[common.Address]CeloPrecompiledContract{
 
 var (
 	PrecompiledAddressesGranite   []common.Address
-	PrecompiledAddressesCel2      []common.Address
```

Comment:
I anyone wonders why PrecompiledAddressesCel2 is different from the other precompiles: The other hard forks are ordered, so the precompiles for each of them is a static full set of precompiles. But the Cel2 hardfork can be added on top of any other hardfork, so we have to dynamically add the Cel2 precompiles to the ones from that hardfork.

Reply:
I guess we should add that as a comment